uk.co.compucorp.civicrm.tasksassignments/scss/civitasks/partials/_spinner.scss

```diff
@@ -4,40 +4,16 @@
   }
 
   .#{$prefix}spinner-cover {
-    @include border-radius($border-radius-base);
-    background-color: rgba(255, 255, 255, 0.5) !important;
     position: absolute;
-    width: 100%;
-    height: 100%;
-    top: 0;
-    left: 0;
+    top: 50%;
```

Comment:
Why do we have dup code? This change already exists in another file in this extension.

Reply:
@igorpavlov  the code in civicrm/civihr#2333 is for `hrjobcontract` extension which has its own version of directive to display and hide spinner. And this code is for T&A extension which also has its own version of directive to achieve the same. 

This PR only focuses on the updating of spinner only. And  yes, it would be great to use the `crm-loading` directive across all CiviHR extensions than having separate directives to handle spinner. And i think this should be the work for another PR.